src/shaderlib/project/project.glsl

```diff
@@ -227,9 +228,9 @@ vec2 project_mercator_(vec2 lnglat) {
 }
 
 vec2 project_position(vec2 position) {
-  // if (projectionMode == PROJECT_LINEAR) {
-  //   return (position + vec2(TILE_SIZE / 2.0)) * projectionScale;
-  // }
+  if (projectionMode == PROJECT_IDENTITY) {
+    return position * projectionScale;
```

Comment:
The original idea was that `projectionScale` would always be 1 in `IDENTITY` - has that changed?